### PR TITLE
Modified the code

### PR DESCRIPTION
**Pull request by Rohit3144**

### CODE REVIEW

**Comment by Rohit3144** [comment]
@prashantpateldixoninfo, please review my code

File: hellofunc.c [hunk 1]
```diff
@@ -3,7 +3,10 @@
 
 void myPrintHelloMake(void) {
 
-  printf("Hello makefiles!\n");
```
Comment (prashantpateldixoninfo) [inline hellofunc.c:6]: Please align the code indentation

**Review by prashantpateldixoninfo — APPROVED** [review]

File: hellofunc.c (hunk 1 above)
Comment (Rohit3144) [reply hellofunc.c:6]: Agree

**Review by prashantpateldixoninfo — APPROVED** [review]

**Comment by prashantpateldixoninfo** [comment]
closing PR

**Pull request closed without merging**